google-cloud-clients/google-cloud-bigquerystorage/pom.xml

```diff
@@ -7,7 +7,7 @@
     <name>Google Cloud Bigquery Storage</name>
     <url>https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-clients/google-cloud-bigquerystorage</url>
     <description>
-       BigQuery Storage API provides fast access to BigQuery table data. 
+        BigQuery Storage API provides fast access to BigQuery table data.
```

Comment:
Was this intentional?

Reply:
no

Reply:
Then please remove this change from your PR

Reply:
already remove from PR